doc/ref_arch/openstack/chapters/chapter03.rst

```diff
@@ -728,166 +741,49 @@ Assumptions and conventions:
-|     | e\ |            |          |       |     |         |        |
-|     | a\ |            |          |       |     |         |        |
-|     | d  |            |          |       |     |         |        |
-|     | a\ |            |          |       |     |         |        |
-|     | c\ |            |          |       |     |         |        |
-|     | r\ |            |          |       |     |         |        |
-|     | o\ |            |          |       |     |         |        |
-|     | s\ |            |          |       |     |         |        |
-|     | s  |            |          |       |     |         |        |
-|     | m\ |            |          |       |     |         |        |
-|     | u\ |            |          |       |     |         |        |
-|     | l\ |            |          |       |     |         |        |
-|     | t\ |            |          |       |     |         |        |
-|     | i\ |            |          |       |     |         |        |
-|     | p\ |            |          |       |     |         |        |
-|     | l\ |            |          |       |     |         |        |
-|     | e  |            |          |       |     |         |        |
-|     | R\ |            |          |       |     |         |        |
-|     | e\ |            |          |       |     |         |        |
-|     | g\ |            |          |       |     |         |        |
-|     | i\ |            |          |       |     |         |        |
-|     | o\ |            |          |       |     |         |        |
-|     | n\ |            |          |       |     |         |        |
-|     | s  |            |          |       |     |         |        |
-+-----+----+------------+----------+-------+-----+---------+--------+
-
-Topology Detail
-~~~~~~~~~~~~~~~
-
-Topology 1 - Local Redundancy
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. list-table:: Cloud Topology: Redundancy Models
+   :widths: 8 15 8 8 8 8 8 17
+   :header-rows: 1
+
+* - Topology Ref
+  - Type
+  - Control Planes
+  - Shared Storage (optional)
+  - Compute AZs
+  - Achievable Service Availability %
+  - Service Multi -region awareness
+  - Notes
+*  - 1
+  - Local Redundancy - workload spread across servers
+  - 1
+  - 1
+  - 1
+  - Variable
+  - Not required
+  - Suitable where only limited local application availability is required e.g. nova anti-affinity
+* - 2
+  - Regional Redundancy - workload spread across AZs
+  - 1
+  - >=2
+  - >=2
+  - >99.n
+  - Not required
+  - Suitable where local application HA is required. Control plane should be distributed across DC failure domains 
+    (assuming layer 2 connectivity) but may be unavailable during up grades
+* - 3
+  - Global Redundancy - workload spread across multiple Regions
+  - >=2
+  - >=2
+  - >=2
+  - >99.nn
+  - Required
+  - Suitable where local and region application HA is required Control plane could be kept available in one site during 
+    upgrades
+
+
+**Topology Overview**
```

Comment:
@pgoyal01 "Topology overview" can be removed here, as it was added above. And the text following the table is an explanation of the table. For me, there is no need of title here.